test/functional/api/cas/installer.py

```diff
@@ -9,68 +9,49 @@
 from tests import conftest
 from core.test_run import TestRun
 
-LOGGER = logging.getLogger(__name__)
-
-opencas_repo_name = "open-cas-linux"
-
 
 def install_opencas():
-    LOGGER.info("Cloning Open CAS repository.")
-    TestRun.executor.run(f"if [ -d {opencas_repo_name} ]; "
-                         f"then rm -rf {opencas_repo_name}; fi")
-    output = TestRun.executor.run(
-        "git clone --recursive https://github.com/Open-CAS/open-cas-linux.git")
-    if output.exit_code != 0:
-        raise Exception(f"Error while cloning repository: {output.stdout}\n{output.stderr}")
-
-    output = TestRun.executor.run(
-        f"cd {opencas_repo_name} && "
-        f"git fetch --all && "
-        f"git fetch --tags {conftest.get_remote()} +refs/pull/*:refs/remotes/origin/pr/*")
-    if output.exit_code != 0:
-        raise Exception(
-            f"Failed to fetch: "
-            f"{output.stdout}\n{output.stderr}")
-
-    output = TestRun.executor.run(f"cd {opencas_repo_name} && "
-                                  f"git checkout {conftest.get_branch()}")
-    if output.exit_code != 0:
-        raise Exception(
-            f"Failed to checkout to {conftest.get_branch()}: {output.stdout}\n{output.stderr}")
+    TestRun.LOGGER.info("Copying Open CAS repository to DUT")
+    TestRun.executor.rsync(
+        f"{TestRun.plugins['opencas']['repo_dir']}/",
+        f"{TestRun.plugins['opencas']['working_dir']}/",
+        delete=True)
 
-    LOGGER.info("Open CAS make and make install.")
+    TestRun.LOGGER.info("Building Open CAS")
     output = TestRun.executor.run(
-        f"cd {opencas_repo_name} && "
-        "git submodule update --init --recursive && "
+        f"cd {TestRun.plugins['opencas']['working_dir']} && "
         "./configure && "
         "make -j")
     if output.exit_code != 0:
         raise Exception(
             f"Make command executed with nonzero status: {output.stdout}\n{output.stderr}")
```

Comment:
please use TestRun.LOGGER.exception instead of raising exception, so output could be visible in test log.

Reply:
Done.